src/helpers/shuffleArrayHelper.ts

```diff
@@ -0,0 +1,11 @@
+export const shuffleArray = <T extends unknown>(array: T[]) => {
+  const newArr = array.slice()
+
+  for (let i = newArr.length - 1; i > 0; i--) {
+    const rand = Math.floor(Math.random() * (i + 1))
```

Comment:
`Math.random()` is unsafe to get always a "real" `random` value. How about to use `[lodash.shuffle](https://www.npmjs.com/package/lodash.shuffle)` (based on [Fisher-Yates shuffle](https://en.wikipedia.org/wiki/Fisher-Yates_shuffle) or maybe another package with less dependencies?

Reply:
yeah. make sense. but i just moved this code outside.
and the second thing is that real randoms do not matter here. next line swaps 2 'whatever elements'. here we dont care about how much random is real

Reply:
`lodash.shuffle` or similar would save us to create this helper and its test here, too.